blocks/commerce-checkout/commerce-checkout.js

```diff
@@ -1,26 +1,22 @@
 /* eslint-disable import/no-unresolved */
+/* eslint-disable import/no-extraneous-dependencies */
+
+// Drop-in Tools
 import { events } from '@dropins/elsie/event-bus.js';
-import { setEndpoint } from '@dropins/elsie/fetch-graphql.js';
-import { initializers } from '@dropins/elsie/initializer.js';
-import * as checkoutApi from '@dropins/storefront-checkout/api.js';
-import Checkout from '@dropins/storefront-checkout/containers/Checkout.js';
-import { render as checkoutRenderer } from '@dropins/storefront-checkout/render.js';
-import { getConfigValue } from '../../scripts/configs.js';
 
-/**
- * Example integration of Checkout Dropin + EDS.
- */
-export default async function decorate(block) {
-  const commerceEndpoint = await getConfigValue('commerce-core-endpoint');
+// Drop-in Providers
+import { render as provider } from '@dropins/storefront-checkout/render.js';
 
-  setEndpoint(commerceEndpoint);
+// Drop-in Containers
+import Checkout from '@dropins/storefront-checkout/containers/Checkout.js';
 
-  initializers.register(checkoutApi.initialize, {
-    // TODO:
-    // cartId: store.getCartId(),
+export default async function decorate(block) {
+  // Listen for order confirmation and redirect to order confirmation page
+  events.on('checkout/order', (data) => {
+    window.location.replace(`/order-confirmation?orderRef=${data.masked_order_id}`);
   });
 
-  checkoutRenderer.render(Checkout, {
+  return provider.render(Checkout, {
```

Comment:
Is this meant to be standard practice for all drop-in rendering?

Reply:
yes